.github/workflows/tests.yaml

```diff
@@ -103,3 +103,63 @@ jobs:
         run: |
           source ${HOME}/.venv/bin/activate
           make test-integrations
+      - name: Run overlay smoke test
```

Comment:
This is a test using two parts with overlays that runs on ci to make sure the basics are working. This test failed with pychroot with multiple process leaks and unmount errors.